.github/workflows/test.yml

```
@@ -0,0 +1,45 @@
+# This workflow will do a clean install of the dependencies and run tests across different versions
+#
+# Replace <track> with the track name
+# Replace <image-name> with an image to run the jobs on
+# Replace <action to setup tooling> with a github action to setup tooling on the image
+# Replace <install dependencies> with a cli command to install the dependencies
+#
+# Find Github Actions to setup tooling here:
+# - https://github.com/actions/?q=setup&type=&language=
+# - https://github.com/actions/starter-workflows/tree/main/ci
+# - https://github.com/marketplace?type=actions&query=setup
+#
+# Requires scripts:
+# - bin/test
+
+name: J / Test
+
+on:
+  push:
+    branches: [main]
+  pull_request: 
```

I would recommend adding this, otherwise PRs will run the tests twice.

```suggestion
  pull_request: 
    branches: [main]
```